neurots/generate/grower.py

```diff
@@ -107,9 +107,10 @@ def __init__(
         self.input_distributions = _load_json(input_distributions)
 
         # Validate parameters and distributions
-        if not skip_validation:
-            self.validate_params()
-            self.validate_distribs()
+        if not skip_preprocessing:
+            self.input_parameters, self.input_distributions = preprocess_inputs(
+                self.input_parameters, self.input_distributions
+            )
 
         # Consistency check between parameters and distributions
```

Comment:
I realize that these checks on `metric` and `method` could also be moved to the validity checkers. But I will do this in another PR to not make this one too big.

Reply:
added!